src/unitful.jl

```diff
@@ -0,0 +1,277 @@
+# previously https://github.com/jw3126/UnitfulRecipes.jl
+# authors: Benoit Pasquier (@briochemc) - Jan Weidner (@jw3126)
```

Comment:
```suggestion
# authors: Benoit Pasquier (@briochemc) - David Gustavsson (@gustaphe) - Jan Weidner (@jw3126)
```

Reply:
Sure, I've copied the contents of https://github.com/jw3126/UnitfulRecipes.jl/blob/master/Project.toml, which weren't up-to-date I guess.